frontend/common/translations.js

```diff
@@ -133,13 +133,14 @@ export default function L (
 
 export function LError (error: Error): {|reportError: any|} {
   let url = `/app/dashboard?modal=UserSettingsModal&tab=application-logs&errorMsg=${encodeURI(error.message)}`
+  const target = !sbp('state/vuex/state').loggedIn ? 'target="_blank"' : ''
   if (!sbp('state/vuex/state').loggedIn) {
     url = 'https://github.com/okTurtles/group-income/issues'
   }
   return {
     reportError: L('"{errorMsg}". You can {a_}report the error{_a}.', {
       errorMsg: error.message,
-      'a_': `<a class="link" target="_blank" href="${url}">`,
+      'a_': `<a class="link" ${target} href="${url}">`,
```

Comment:
I don't think this change does anything useful, should be undone.

There is a related issue #1945, but this change doesn't address that.